examples/pubsub.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+
+	"../nodego"
+)
+
+func main() {
+	flag.Parse()
+
+	http.HandleFunc(nodego.PubSubTrigger, nodego.EventHandler(func(event *nodego.FunctionEvent) error {
+		msg, err := event.PubSubMessage()
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf("Your message: %s\n", msg.Data)
```

Comment:
I think STDOUT goes into a black hole on GCF. Maybe it would be better to log it?